content/docs/reference/rich-text-usage/markdown-tables.mdx

```diff
@@ -46,7 +46,7 @@ const MyTable = props => <table>
 
 ## Showcase
 
-![](https://res.cloudinary.com/forestry-demo/image/upload/v1726549733/tina-io/docs/2024-09-13_15-59-11_1_y2elbn.gif)
+<WebmEmbed embedSrc="public/img/docs/reference/cells-and-rows.webm"/>
```

Comment:
**issue**: Images not shown due to 'public' prefix.

Reply:
Resolved